### ISSUE TITLE
Duplicated functions in dist/react-data-grid.js

### ISSUE THREAD

**Issue by MalcolmDwyer** [adazzle/react-data-grid#57]
My webpack build includes a preLoader run through JSCS.  It errors out on react-data-grid.js because render() and renderStatusIcon() functions both appear twice in the EditorContainer component (around line 4080).  They appear to be identical, so probably no harm other than having lint checkers barf.

**Comment by bakesteve** [comment]
odd - is that off latest master (build from source) or npm? we have that only once, and @ line 3413

**Comment by MalcolmDwyer** [comment]
Hi... sorry, I should have said earlier: the issue exists on 0.13.0-alpha1
I know there are other issues with that build.  This was the first one I ran into though.

```bash
$ grep react-data-grid package.json 
    "react-data-grid": "0.13.0-alpha1"

$ npm install --save react-data-grid
 

$ grep -n -C 5 'renderStatusIcon:' node_modules/react-data-grid/dist/react-data-grid.js 
4012-	    return joinClasses({
4013-	      'has-error': this.state.isInvalid === true
4014-	    });
4015-	  },
4016-
4017:	  renderStatusIcon: function renderStatusIcon() {
4018-	    if (this.state.isInvalid === true) {
4019-	      return React.createElement('span', { className: 'glyphicon glyphicon-remove form-control-feedback' });
4020-	    }
4021-	  },
4022-
--
--
4072-	    if (!this.changeCommitted && !this.hasEscapeBeenPressed()) {
4073-	      this.commit({ key: 'Enter' });
4074-	    }
4075-	  },
4076-
4077:	  renderStatusIcon: function renderStatusIcon() {
4078-	    if (this.state.isInvalid === true) {
4079-	      return React.createElement('span', { className: 'glyphicon glyphicon-remove form-control-feedback' });
4080-	    }
4081-	  },
4082-
```

Both of those renderStatusIcon functions are in the same block... and render() is repeated there as well.

- Thanks

**Comment by malonecj** [comment]
Hi, this commit should fix the problem
https://github.com/adazzle/react-data-grid/commit/bdcc1766b12f14a7ee387222006ed5bce1027b48

I have published a new version for the React 13 upgrade 0.13.0-alpha2
